napari/layers/_tests/test_layer_actions.py

```diff
@@ -11,12 +11,13 @@
 )
 
 
-def test_duplicate_layers():
+@pytest.mark.parametrize('layer_type', [Points, Shapes])
+def test_duplicate_layers(layer_type):
```

Comment:
Did this test check if the duplicate layer is disabled for the empty layer? If yes, then the name should be updated. 

Also, checking the length of callbacks is a dummy. There should be a check for given callback absence.

Reply:
@dalthviz I need a response to this question before making a decision about approval.

Reply:
Here I only parametrized an already existing test that was checking only the duplication of a `Points` layer instance to also check using the `Shapes` layer type. There is no check for the layers actions that appear on the context menu. Should I try to further modify the test to check the state of the layer actions that appear in the context menu?

Reply:
Ok, I see

So there is no direct test of `_empty_shapes_layer_selected`?

Reply:
Nope